### ISSUE TITLE
Logo for JoinMarket

### ISSUE THREAD

**Issue by ashmoran** [JoinMarket-Org/joinmarket#381]
I was reading the wiki page [What can I do for JoinMarket?](https://github.com/JoinMarket-Org/joinmarket/wiki/What-can-I-do-for-JoinMarket%3F), and point 3 is "Create a logo to represent JoinMarket".

I had an idea. I could commission some drafts on [Fiverr](https://www.fiverr.com/categories/graphics-design/creative-logo-design/), post them here, and if you like any of them have them developed. We had the logo for the Bitcoin Manchester group designed locally, it took a bit of going backwards and forwards, but we ended up with something we liked. This way shouldn't cost too much and I don't think it'll take a huge amount of my time in terms of hours invested, although it might take a few weeks calendar time to get anything suitable.

If anyone thinks this is a good idea, please nod and I'll make a start. Suggestions welcome.

**Comment by AdamISZ** [comment]
We had a chat about this on IRC (more than once, I guess).

A blender seemed like one interesting option. See the last few comments here: https://github.com/JoinMarket-Org/joinmarket/issues/244#issuecomment-175269448

As for using Fiverr, I don't know. It seems like a good idea I guess, I'm rather agnostic. Anyone else opinions? @chris-belcher @adlai ?

**Comment by chris-belcher** [comment]
I'm also agnostic about Fiverr, although I note it costs so for it to happen someone would have to put up the money.

**Comment by ashmoran** [comment]
> I'm also agnostic about Fiverr, although I note it costs so for it to happen someone would have to put up the money.

That's what I was offering to do :) I think JM is a valuable addition to Bitcoin. I'm not likely to be any help with the code, but I saw this and thought I'd suggest it instead.

I have no design skill whatsoever, but I don't mind throwing $20-30 or so at it to see if produces any usable ideas. If it turns up a pile of crap that nobody likes, I won't mind. That's what you expect for $5. But there's a chance it might produce something worth developing.

**Comment by chris-belcher** [comment]
Ok I didn't get that part. : )

It's worth a go I guess.

**Comment by adlai** [comment]
Couldn't we set aside some portion of the accumulated donations for this?

**Comment by ashmoran** [comment]
I put up a request on Fiverr earlier today and I've had 3 offers come back in so far. Two of them even have feedback, so I'm quite optimistic about getting some files back.

@adlai No need to worry about repaying me, I'm hoping the total cost won't be that great. Until I lose as much on Fiverr as I did on Litecoin, the logo design is on me. Sadly Bitcoin doesn't have a native governance system for this sort of thing; Dash has a development budget stream and a voting system. For now, graphic design work can come out of the [Bitcoin Manchester](http://www.bitcoinmanchester.org.uk) pizza fund, as all the venture capital money in Bitcoin is currently being spent preventing excessive network growth and ensuring unconfirmed transactions are not abused for payments.

In any case I'm quite enjoying the variety of a mundane management task in place of programming. I'm already considering growing my hair more pointy. I might break the South Park Avatar Generator if I do, though.

**Comment by DavidVorick** [comment]
I like the idea of a shape like this: `><`, basically two inputs, crossing/merging in the middle, and then appearing as two outputs. I'd draw one but I don't have any drawing programs on my computer. Maybe I'll draw some on paper.

**Comment by ashmoran** [comment]
Well that was an exercise in frustration and futility. 

Initially I put up an advert to see who would respond. One designer pulled out because he couldn't understand the spec. A couple didn't understand what a file extension or text editor was after I made the mistake of sending the spec as a .markdown file. One did not notice that I sent a spec. One tried to withdraw her work after I paid her because she didn't like the review of her designs which incorporate exactly 0 of the concepts, and she pestered me so much I asked support to stop the harassment (this may partly explain why the ratings on Fiverr are so skewed to 5*). One designer can't spell the word "designed". Thinking that I might have better luck with the ones too busy to scour for adverts, I contacted a few directly. One of these I commissioned for 3 logos (many designers voluntarily produced multiple designs) because they had some good samples, and came back with one logo rearranged into 3 formats. Out of all of them, I think my favourite submission was the one that incorporated the £, $ and ¥ signs, but no trace of Bitcoin whatsoever (this is the only one I rejected).

Anyway, the experiment has produced a few drafts. Production quality ranges from fairly decent to "just discovered WordArt". Incorporation of JM concepts ranges from one or two to "did not read the spec". My list of suggested concepts was: "Bitcoin", "transactions", "mixing", "confidentiality", "privacy", "anonymity", "fungibility (all money is equal)", "marketplace (anyone is free to make or take offers)". I suspect language is a problem: I spoke to only two designers with proficient/native English, they were the only two who explained what concepts they were was using and why. I'm not convinced many of the rest looked up the words they didn't know. What's funny is the number of people who added padlocks, despite the fact I never mentioned the word "security" once (one of the two designers who explained their thinking said it was to represent privacy). Only two people asked for clarification: one of the designers who explained his thinking, and the guy who didn't understand anything at all.

I've uploaded the files as an Imgur album (unfortunately some transparent PNGs don't display well on Imgur's black background): http://imgur.com/a/im7Lu

If anyone likes any of these, I could get one polished up, or go back and ask for the vector source. If not, I knew beforehand it was a crapshoot, and I wanted to do it anyway as a learning exercise.

In summary, this exercise has created: 43 logo drafts, 13 Fiverr reviews, and 1 case study on why outsourcing won't save you money.

**Comment by chris-belcher** [comment]
IMO I personally don't much like any of them. Sorry. Thanks for doing this.
The ones with the dark people silhouettes allude to criminal activity which is not what we want to be associated with. I guess if I absolutely had to pick one it would be ebilal_2.

**Comment by ashmoran** [comment]
That's fine, I thought it was at best 50:50 whether it would produce anything acceptable. Personally I quite liked the concept in wrightdesigner_2, although it could possibly be mistaken for a dating site, and the font is possibly the only one in existence worse than Comic Sans.

I'll let someone else have a go :)

**Comment by AdamISZ** [comment]
Thanks @ashmoran . I quite like the logotina_* ones (at least, i like the jm part). It doesn't have special meaning but is reasonably visually distinctive. I don't really like the ebilal_* ones including 2. Re: the wrightdesigner_2, I don't think a padlock really works here as a concept.

Oh well, I guess we could be here all day giving opinions :) Much appreciate the effort!

**Comment by ashmoran** [comment]
@AdamISZ The concept I liked in wrightdesigner_2 was the two people doing the waltz/tango in the o of Join. I agree about the padlock not working… I have no idea why so many people added a padlock/safe, despite me never mentioning anything about security. It never occurred to me to list what *not* to include. Lesson learnt.

I quite like logotina's typography too (they look better against a while background), and I think creative_boost's is not far off. Unfortunately neither included any JM concepts, in fact they didn't even add a stray padlock.

> Oh well, I guess we could be here all day giving opinions :)

A little bikeshedding is highly therapeutic :D

**Comment by adlai** [comment]
> A little bikeshedding is highly therapeutic :D
Or is talk therapy all just bikeshedding?

My initial reaction is a capital NO to the one that says "100% GUARANTEED", because JoinMarket comes with a zero percent privacy guarantee, and an unknown percent privacy hopefully-maybe-if-you-try-hard-enough.

My subsequent reaction is that I'm strongly in favor of using [illusory contours](https://en.wikipedia.org/wiki/Gestalt_psychology#Reification) in our logo. My favorite version of this from the album @ashmoran linked is [elibal_1](https://i.imgur.com/BlAbZMn.png), with the added bonus that it is simple enough to work as an icon for the GUI app (or favicon for websites).

**Comment by azzuron** [comment]
I know i'm a nobody, :) but, ebilal_2 with some interactions i think could work really well. its a difficult concept to visually represent i think without it being a bunch of bubbles or boxes linked by lines. Padlocks are often used for security, but also privacy. just google image search privacy and you will see a number of padlocks. 

My sister is a pretty accomplished graphic designer. I could seek her input on some of this and see if she has any thoughts on it.

**Comment by chris-belcher** [comment]
On reddit this logo has been used for about a month: https://www.reddit.com/r/joinmarket/comments/55o6ht/newest_final_logo_revision_thoughts/

**Comment by azzuron** [comment]
Oh i see. OK. well if the logo issue is done, perhaps we can close this issue.
